report/src/main/java/com/megaease/easeagent/report/sender/okhttp/HttpSender.java

```diff
@@ -32,6 +32,7 @@
 import okio.BufferedSink;
 import okio.GzipSink;
 import okio.Okio;
+import org.slf4j.Logger;
```

Comment:
According to the Logger usage agreement we provide, you need to replace the Logger with our own com.megaease.easeagent.log4j2.Logger

Reply:
let's make usage of slf4j logger has the completely same ability as com.megaease.easeagent.log4j2.Logger, making the usage of the logger in Easeagent easier.